### ISSUE TITLE
Allow clients to download optimized images

### ISSUE THREAD

**Issue by sampullman** [samatechtw/image-api#50]
Return an `accessUrl` in the get-job request once the job is complete.

If the original create-job request contained an `uploadUrl`, return that. Otherwise, expose the tmp output file. The easiest way is probably to configure nginx to serve `/tmp/image-api/`, although it's not ideal from a security perspective.

Depending on how the server is set up, tmp files will be deleted after 30 days. This is nice to have though, so we don't have to worry about cleaning up old files ourselves yet.
